### ISSUE TITLE
Spotify Connect stopped working recently

### ISSUE THREAD

**Issue by chrisgrieser** [librespot-org/librespot#1123]
**Describe the bug**
I am a user of `spotify-tui` and noticed that since today/yesterday, it appears that Spotify Connect is not working. I [reported the issue there](https://github.com/Rigellute/spotify-tui/issues/1072) and was told that this seems to be an upstream issue with Spotify API changes which I should report here.

**To reproduce**
1. Install spotify-tui
2. setup for making spotify-tui work with Spotify Connect
3. start the tui, press `d` (for device) and select a Spotify Connect device (in my case an Alexa speaker)
4. Result is this error

![CleanShot 2023-02-22 at 19 35 38](https://user-images.githubusercontent.com/73286100/220725105-87e89a60-6de6-46e2-a95d-a201f2ca776f.png)

**Host (what you are running `librespot` on):**
- macOS 13.2 (M1)
- spotify-tui 0.25.0

**Comment by kingosticks** [comment]
Imagine we have never used spotify-tui and don't want to start. What is `d` doing? Device discovery? Do other devices work? Can you capture getInfo response your Alexa device provided?

**Comment by chrisgrieser** [comment]
> What is `d` doing? Device discovery?

yes, just selecting devices

> Do other devices work?

controlling other devices (web player, desktop client) via spotify-tui works fine. Controlling Alexa devices via the official spotify client, phone or web player works fine. 

Controlling alexa devices via non-first party clients does not work (tried spotify-tui and spotify-player, I am not aware of any other that can use Spotify Connect).

>  Also assume we don't have an Alexa device, can you capture the getInfo response your Alexa device provided?

I don't know how to do that, tbh – I have never used and heard of librespot until I was told to report the issue here an hour ago.

**Comment by kingosticks** [comment]
Fair enough. Hopefully someone at spotify-tui can fill in the gaps of how exactly they are using librespot and how to get logs.

**Comment by Neburb** [comment]
Hey, I found this while searching through google, and I think I can add my two cents here.
Are you people using the spotify web API under the hood for `librespot`? I have the same issue while using the plain spotify web API.
Using any other device (Phone, PC app, etc) while calling `play` works as a device_id, but using an Alexa echo device ID doesn't. I don't know if it is an issue with Alexa or Spotify web API, but I started having this issue a day, or a couple of days ago.
